framework/db/Connection.php

```diff
@@ -137,19 +137,19 @@ class Connection extends Component
     /**
      * @event Event an event that is triggered after a DB connection is established
      */
-    const EVENT_AFTER_OPEN = 'afterOpen';
+    const EVENT_AFTER_OPEN = 'db.connect.after';
```

Comment:
Should it be `db.connection.afteropen`?

Reply:
That is hard to tell: at the first edition this was `db.connection.open.after` assuming that it is possible for `db.connection.open.before` to exist.
I would rather avoid having chain part consisting of two words like 'afteropen' - word are better to be separated.

Reply:
`db.connection.open.after` is a better name.

Reply:
done